app/templates/merchant/view_listings.html

```diff
@@ -109,6 +140,17 @@
                 updateCartQuantity(listingID, newQuantity);
                 updateCartFrontend(listingID, newQuantity);
             });
+
+        })
+
+
+        $('.toggle-cart').click(function(event) {
+            var toggleCartIcon = $( this );
+            var inCart = toggleCartIcon.parent().data("incart");
+            var listingId = toggleCartIcon.parent().data("id");
+            var newValue = inCart === true ? false : true;
+            var newQuantity = newValue === true ? 1 : 0;
+            updateCartQuantity(listingId, newQuantity);
```

Comment:
We can get rid of this function, since we already moved the logic into the `document.ready` above